js/cards.js

```diff
@@ -33,6 +36,10 @@ export async function defineCardComponent() {
       this.data = cards_data.find(
         (card) => card["Collector Number"] == this.getAttribute("card-id")
       );
+      if (this.data["Rarity Folder"] === "Element") {
+        this.data = Object.assign({}, this.data);
+        this.data["Collector Number"] = "000";
+      }
```

Comment:
I assume this was added before the Elements had a collector number in the CSV. Is this still needed now that the latest CSV has collector number 000-1, 000-2 etc?

Reply:
This was added after that. By doing this, the system internally manages element cards with their assigned IDs but uses 000 for everything visually related. I added a comment; the Object.assign call copies the data object by value so it can be changed without it affecting its ID on the cards_data array. (since objects are passed by reference by default in JS)